src/attributes.jl

```diff
@@ -1337,17 +1337,17 @@ struct ConstraintDualStart <: AbstractConstraintAttribute end
 A constraint attribute for the assignment to some constraint's primal value(s)
 in result `result_index`. If `result_index` is omitted, it is 1 by default.
 
-See [`ResultCount`](@ref) for information on how the results are ordered.
+If the constraint is `f(x) in S`, then the `ConstraintPrimal` is the value of
+`f`, evaluated at the correspondng [`VariablePrimal`](@ref) solution.
 
-## Example
+See [`ResultCount`](@ref) for information on how the results are ordered.
 
-Given a constraint `function-in-set`, the `ConstraintPrimal` is the value of the
-function evaluated at the primal solution of the variables.
+## Conic solvers
 
-For example, given the constraint
-`ScalarAffineFunction([x,y], [1, 2], 3)`-in-`Interval(0, 20)` and a primal
-solution of `(x,y) = (4,5)`, the `ConstraintPrimal` solution of the constraint
-is `1 * 4 + 2 * 5 + 3 = 17`.
+Some conic solvers reformulate `b - Ax in K` to `s = b - Ax, s in K`. These
+solvers should return the value of `s` for `ConstraintPrimal`, rather than
```

Comment:
I think it should be formulated as a new rule but rather as an example that the definition is flexible, e.g. that the solver may return something that is not exactly `f(x)` if it serves as a slack.
So the only think you know is that `ConstraintPrimal` and `ConstraintPrimalStart` correspond to the same thing and that `ConstraintPrimal` should be close to `b - Ax` for a feasible solution

Reply:
How's this now